lib/rules/no-side-effects-wait-for.ts

```diff
@@ -60,6 +62,9 @@ export default ESLintUtils.RuleCreator(getDocsUrl)<Options, MessageIds>({
     return {
       [`${WAIT_EXPRESSION_QUERY} > ArrowFunctionExpression > BlockStatement`]: reportSideEffects,
       [`${WAIT_EXPRESSION_QUERY} > FunctionExpression > BlockStatement`]: reportSideEffects,
+      ImportDeclaration(node: TSESTree.ImportDeclaration) {
+        isImportingTestingLibrary = /testing-library/g.test(node.source.value as string);
```

Comment:
You already have hasTestingLibraryImportModule util for checking this. Could you use that one instead please?